src/css/header.css

```diff
@@ -0,0 +1,39 @@
+#header {
+  display: -webkit-box;
+  display: flex;
+  height: 5em;
+  max-height: 5em;
+  -ms-flex-line-pack: center;
+  align-content: center;
+  padding: 2em 2em 6em 6em;
+}
```

Comment:
* The navigation bar has too much padding. I suggest you reduce the padding or remove it to provide a better look and feel
* I'd please like to know why you use max-height on the nav. I suggest you use min-width instead to prevent nav contents from overflowing as you move to smaller screens.

Reply:
Made changes 👍